packages/model/Project.ts

```diff
@@ -22,19 +22,33 @@ export interface Project extends RdfResource {
   creator: Person
   label: string
   jobCollection: Link<JobCollection>
-  cubeIdentifier: string
   maintainer: Link<Organization>
   publishedRevision: number
+  sourceKind: NamedNode
+}
+
+export interface CsvProject extends Project {
+  cubeIdentifier: string
+}
+
+export interface ImportProject extends Project {
+  importCube: NamedNode
+  importFromGraph: NamedNode | undefined
+  importFromEndpoint: NamedNode
```

Comment:
Don't particularly like the names. Maybe `sourceCube` etc?

Reply:
I like `sourceCube`, `sourceGraph` and `sourceEndpoint`